### ISSUE TITLE
alpine not working

### ISSUE THREAD

**Issue by lindt** [soulshake/clink#4]
could not describe any further now. but want to keep the other issue clean from this discussion here.
even "ubuntu" is not working anymore on master.

will investigate on this, once I'm able to run "ubuntu" again.

expectation:

```
> alpine /bin/sh
```

should open a docker container based on alpine.

**Comment by soulshake** [comment]
I can't reproduce. Can you provide more details? You can `export CLINK_VERBOSITY=1` for more verbose output.

**Comment by lindt** [comment]
/bin/sh is on my system a link to /bin/bash
opensuse default, i guess.

once i clink 'ubuntu /bin/sh', then '/bin' is detected as volume and mounted into the container. (--volume /bin:/bin)
on ubuntu there is a bash, so the link is working. on alpine there is no bash, so it's not working.

Why do I need to mount my args as volumes. with something under ~ it may be useful, but system directories?

Sorry for file these issues. My usecases seem different than yours. I normally just stdin/out to docker on command line. also wouldn't need all the devices.

**Comment by soulshake** [comment]
Good point. We should detect if it's an executable rather than just checking if it's a file that exists. I'm working on other things right now but will try to take a look at some point.
